ui/nuxeo-filters-behavior.js

```diff
@@ -273,4 +273,23 @@ export const FiltersBehavior = {
     const processes = document.contextParameters.runnableWorkflows;
     return processes && processes.length > 0;
   },
+
+  /**
+   * Checks if the given document or its main content, can be deleted by the given user
+   */
+  canDelete(document, user) {
+    const underRetentionOrLegalHold = this.isUnderRetentionOrLegalHold(document);
+    const hasRemovePermission = this.hasPermission(document, 'Remove');
+    if (!underRetentionOrLegalHold && hasRemovePermission) {
+      return true;
+    }
+    return (
+      (this.hasAdministrationPermissions(user) || this.hasPermission(document, 'RemoveRecords')) &&
```

Comment:
Shouldn't we check for the 'Remove' permission here as well? Or does the RemoveRecords permission already includes the 'Remove' one? Where can I find the permission's declaration, btw?